utils/rpms/daos.spec

```diff
@@ -34,7 +34,6 @@ BuildRequires: scons >= 2.4
 BuildRequires: libfabric-devel >= %{libfabric_version}
 BuildRequires: mercury-devel >= %{mercury_version}
 %if (0%{?rhel} < 8) || (0%{?suse_version} > 0)
-BuildRequires: openpa-devel
```

Comment:
don't think we need libpsm2-devel either

Reply:
ok I think in that case we should also remove libpsm_infinipath1 because I really can't see why it would ever be needed.

Reply:
might be better to do it separately though since there's a PSM2 dependency